### PR TITLE
[ADD] accounting (localizations): list of supported countries

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

File: accounting/fiscal_localizations/overview/localizations_list.rst
Comment (cdu-odoo) [inline accounting/fiscal_localizations/overview/localizations_list.rst:5]:
```suggestion
**Odoo Accounting** can be used in many countries out of the box by installing the appropriate module.
```

File: accounting/fiscal_localizations/overview/localizations_list.rst
Comment (cdu-odoo) [inline accounting/fiscal_localizations/overview/localizations_list.rst:124]: Alphabetical order? Easier to find the information nope? :)

Comment (jcs-odoo) [reply accounting/fiscal_localizations/overview/localizations_list.rst:124]: .... of course ^^ you're right.

**Comment by jcs-odoo** [comment]
@cdu-odoo 
alphabetical order done
+ fix: missing space
+ fix: swedish groupment
+ fix: something but I don't remember what :p

**Review by cdu-odoo — APPROVED** [review]
Niiicee 👍

**Comment by jcs-odoo** [comment]
@jubodoo 
could you approve this pr that is waiting for your approval since august please? :)

**Review by jubodoo — APPROVED** [review]

**Pull request merged**
